events/beam-robotics.html

```diff
@@ -0,0 +1,166 @@
+---
+layout: default
+title: Robotics BEAM page
+permalink: /beam-robotics
+---
+<div class="beam-top-section">
+  <div class="grid-col">
+    <h1 class="title page-title"> Robotics Fun with BEAM </h1>
+    <p class="title text-2x no-margin">
+        <a class="tla-link" href="https://beam.ucla.edu/" target="_blank" rel="noopener noreferrer">
+       BEAM at UCLA</a> X ACM Teach LA
+   </p>
+  </div>
+</div>
+
+<p>
+    We partnered with <a class="tla-link" href="https://beam.ucla.edu/" target= "_blank" rel="noopener noreferrer"><b>BEAM at UCLA</b></a> to teach 4th and 5th graders some cool <b>robotics</b>! We have <b>5 weeks</b> of <b>hour long lessons</b> and you can follow along what we did here!
+</p>
+
+<hr class="divider" />
+
+<div class="secondary-top-section">
```

Comment:
this doesn't exist in the css?
use `my-3` instead or something

propagate this change (you proabbly dont need the beam button scss then)

Reply:
can you check that the other scss elements that you added earlier are properly removed from the html files?